### PR TITLE
renaming fields

### PR DESCRIPTION
**Pull request by fabricio-aguiar**
closes #5454
closes #5457
https://pulp.plan.io/issues/5454
https://pulp.plan.io/issues/5457

Required PR: https://github.com/pulp/pulpcore-plugin/pull/137

### CODE REVIEW

File: docs/workflows/publish-host.rst [hunk 1]
```diff
@@ -13,9 +13,9 @@ Create a Publication
 Publication GET Response (after task is complete)::
 
     {
-        "_created": "2019-05-16T19:28:42.971611Z",
+        "pulp_created": "2019-05-16T19:28:42.971611Z",
         "_href": "/pulp/api/v3/publications/file/file/7d5440f6-202c-4e71-ace2-14c534f6df9e/",
```
Comment (dralley) [inline docs/workflows/publish-host.rst]: I think it was decided to rename ```_href``` as well

File: docs/workflows/sync.rst [hunk 2]
```diff
@@ -10,7 +10,7 @@ Create a repository ``foo``
 Repository GET Response::
 
     {
-        "_created": "2019-05-16T19:23:55.224096Z",
+        "pulp_created": "2019-05-16T19:23:55.224096Z",
         "_href": "/pulp/api/v3/repositories/680f18e7-0513-461f-b067-436b03285e4c/",
         "_latest_version_href": null,
```
Comment (dralley) [inline docs/workflows/sync.rst]: I don't think changing these (```_latest_version_href```, ```_versions_href```) was explicitly discussed but we might want to do so for the same reasons.  @dkliban ?

File: docs/workflows/sync.rst [hunk 3]
```diff
@@ -31,10 +31,10 @@ Create a new remote ``bar``
 Remote GET Response::
 
     {
-        "_created": "2019-05-16T19:23:56.771326Z",
+        "pulp_created": "2019-05-16T19:23:56.771326Z",
         "_href": "/pulp/api/v3/remotes/file/file/e682efef-3974-4366-aece-a333bfaec9f3/",
         "_last_updated": "2019-05-16T19:23:56.771341Z",
```
Comment (dralley) [inline docs/workflows/sync.rst]: ```_last_updated``` need to be fixed here

File: docs/workflows/publish-host.rst (hunk 1 above)
Comment (fabricio-aguiar) [reply docs/workflows/publish-host.rst]: oh, I forgot it, so I have to change https://github.com/pulp/pulpcore/pull/317 also

File: docs/workflows/sync.rst (hunk 2 above)
Comment (dralley) [reply docs/workflows/sync.rst]: Discussion continuing here: https://pulp.plan.io/issues/5457#note-12

**Comment by daviddavis** [comment]
This needs to be rebased.

**Pull request merged**